### ISSUE TITLE
vlc (and avidemux) plf won't install (Bugzilla Bug 893)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#893]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 893
Date: 2014-08-08 10:29:12 +0000
From: @jclvanier
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   @benbullard79, @itchka, @robxu9, @tpgxyz

Depends on:   #827
Last updated: 2014-09-13 18:23:12 +0000

## Comment 6770
Date: 2014-08-08 10:29:12 +0000
From: @jclvanier

vlc and avidemux plf wont install because libx264_133 (or lib64x264_133) is missing. See bug #827.
If you install libx264_133 from the 2013 repositories, vlc and avidemux will install. If libx264_142 is also installed, you can remove libx264_133 with the option --nodeps and vlc/avidemux will keep working.

## Comment 6871
Date: 2014-08-20 10:02:13 +0000
From: @itchka

Can we copy the libx264_133 to the 2014 repos and then rebuild the metatdata?

## Comment 6923
Date: 2014-08-29 20:08:20 +0000
From: @tpgxyz

It is already done, waiting for confiramtion. See bug #827

## Comment 7047
Date: 2014-09-13 17:54:07 +0000
From: @benbullard79

Today:

# urpmi libx264_133
No package named libx264_133

## Comment 7048
Date: 2014-09-13 18:19:08 +0000
From: @benbullard79

I was wrong I'm in x86_64. Could someone confirm if this bug is fixed.

## Comment 7051
Date: 2014-09-13 18:23:12 +0000
From: @robxu9

This fix has been pushed to the 2014.0 repositories. If you still experience problems, please reopen this bug.
